app/src/query.rs

```diff
@@ -39,6 +39,15 @@ impl<'app, D: Dependencies<'app>> ApplicationReader<'app, D> {
         self.genesis_config.chain_id
     }
 
+    pub fn client_version(&self) -> String {
+        format!(
+            "op-move/{}/{}/{}",
```

Comment:
This is what op-move vs OP Mainnet returns:
```
op-move/0.1.0/aarch64-apple-darwin/rust_1.85.1
Geth/v1.101503.4-stable-2b9abb39/linux-amd64/go1.24.2
```
| | name | version | target | compiler |
| -- | -- | -- | -- | -- |
| **Umi** | op-move | 0.1.0 | aarch64-apple-darwin | rust_1.85.1 |
| **OP** | Geth | v1.101503.4-stable-2b9abb39 | linux-amd64 | go1.24.2 |

We don't need to fix them cause I don't know how useful this RPC is, but here are the differences I notice:
1. The version needs a `v` prefixed, the rest is OK
2. Target triplet has the `OS` first then the `arch`
3. No underscore needed after `rust`